libraries/stdlib/src/generated/_Ranges.kt

```diff
@@ -677,6 +677,7 @@ public fun Short.coerceAtLeast(minimumValue: Short): Short {
 
 /**
  * Ensures that this value is not less than the specified [minimumValue].
+ * @sample samples.number.Int.coerceAtLeast
```

Comment:
These functions are generated from a template. I think it'd be ok to provide the single sample for all overloads, so this sample reference can be placed in that template. You can find this template here: https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-stdlib-gen/src/templates/Comparables.kt#L27